.gitignore

```diff
@@ -36,6 +36,9 @@ reports
 zeppelin-web/node_modules
 zeppelin-web/dist
 zeppelin-web/.tmp
+zeppelin-web/src/fonts/Roboto*
+zeppelin-web/src/fonts/Source-Code-Pro*
+zeppelin-web/src/fonts/Patua-One*
```

Comment:
now ci fails with 

```
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.11:check (verify.rat) on project zeppelin-web: Too many files with unapproved license: 6 See RAT report in: /home/travis/build/apache/zeppelin/zeppelin-web/target/rat.txt -> [Help 1]
```

and zeppelin-web/target/rat.txt is

```
6 Unknown Licenses

*******************************

Unapproved licenses:

  src/fonts/Patua-One.css
  src/fonts/Patua-One.svg
  src/fonts/Roboto.css
  src/fonts/Roboto.svg
  src/fonts/Source-Code-Pro.css
  src/fonts/Source-Code-Pro.svg
```

And i think above 3 lines can be moved from `.gitignore` to [apache-rat-plugin exclude section](https://github.com/apache/zeppelin/blob/master/zeppelin-web/pom.xml#L78)